src/configurator.js

```diff
@@ -0,0 +1,35 @@
+
+/**
+ * The configurator enables you to create a configuration function quickly
+ *
+ * @param {Config} Config - A plugins class method.
+ * @param {string} [configureMethod='configure'] - the name of the configure
+ * method can be changed in case it is not configure
+ *
+ * @return {<config>} instance of the <Config>
+ *
+ * @example
+ *
+ * import {Config} from './config';
+ * export {configure: configurator(Config);
+ */
+export function configurator(Config, configureMethod = 'configure') {
+  /**
+   * configure aurelia-config
+   * @param  {Aurelia}         aurelia           The aurelia instance
+   * @param  {function|Object} configOrConfigure The configuration object or function
+   * @return {Config}                            The aurelia-config Config instance
+   */
+  return function configure(aurelia, configOrConfigure = {}) {
+    let config = aurelia.container.get(Config);
+
+    if (typeof configOrConfigure === 'function') {
+      configOrConfigure(config);
+    } else {
+      config[configureMethod](configOrConfigure);
```

Comment:
Maybe require one to say what method to use instead of 'configure' as default, and if not said, it throws a descriptive error when an object is passed which says.

```js
getLogger('aurelia-config').error('configureMethod argument is required when wanting to pass objects for configuration', configOrConfigure);
```
Also logging the config object so developer knows for what config it is showing this error.